api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -252,7 +252,6 @@ private boolean isModuleEnabledInEnvironment(AccessModule module) {
     final AccessConfig accessConfig = configProvider.get().access;
 
     return switch (module) {
-      case ERA_COMMONS -> accessConfig.enableEraCommons;
```

Comment:
this will make ERA_COMMONS **always** required, instead of what we want: **never** required.  Let's hard-code this to false instead.

Also: we will need to remove the ERA_COMMONS module itself in later PRs.

Reply:
oh, nice catch! I'm gonna try removing `ERA_COMMONS` from module if it's not too involving

Reply:
I realize I'll have to update contract for removing `ERA_COMMONS` from access module, which will require UI removing it first..so I'll do the easier change for now. And we can continue to `ERA_COMMONS` access module later